Tests/NostrSDKTests/Fixtures/reaction.json

```diff
@@ -0,0 +1,19 @@
+{
+    "content": "🤙",
+    "created_at": 1689029084,
+    "id": "8a3217770794fabe89adac500dcd5d38966d3ba3cb83fabc97b58135980f76cd",
+    "kind": 7,
+    "pubkey": "2779f3d9f42c7dee17f0e6bcdcf89a8f9d592d19e3b1bbd27ef1cffd1a7f98d1",
+    "sig": "c0dea5d4612d834e13e0dcfeff71a345f761d868bf27fd5e3fe521b76872d5da3db05375f8739a4bad86189d63720187c08170827990b113b477437f17e4a906",
+    "tags":
+    [
+        [
+            "e",
+            "62dcc905c282dd712bbe6b47d2e40feb333f8a0c39899617f4ca37337199ede0"
```

Comment:
On an unrelated note, if I remove this value, we get a crash. We should be more resilient against malicious or poorly coded clients adding "malformed" tags.

Reply:
Do you know which test fails?

Reply:
The event decoding ones. The Tag initializer requires there to be a tag name and value.

Reply:
Absolutely. I haven't even started to try to think of all the ways these could go wrong. But yeah we definitely should add tests for all kinds of nonsense we could see from relays or clients.

Reply:
https://github.com/nostr-sdk/nostr-sdk-ios/issues/66